pkg/config/lima_config_applier.go

```diff
@@ -16,7 +16,32 @@ import (
 	"github.com/runfinch/finch/pkg/system"
 )
 
-const userModeEmulationProvisioningScriptHeader = "# cross-arch tools"
+const (
+	sociVersion                              = "0.3.0"
```

Comment:
maybe not in this PR but we should think about / track automatically updating this version in CI somewhere

Reply:
created issue #526 for this comment